flags/src/main/java/com/yahoo/vespa/flags/Flags.java

```diff
@@ -164,7 +164,7 @@
             "use-old-metrics-checks", true,
             "Whether to use old metrics checks",
             "Takes effect on restart of Docker container",
```

Comment:
```suggestion
             "Takes effect on next host admin tick",
```

Reply:
Yeah, looks like we want this on every tick